src/stylesheets/elements/_media.scss

```diff
@@ -58,7 +66,7 @@ figcaption {
   @include css-animation(.8s, linear, 0s, infinite, spinner);
 }
 
-.video-wrap, // ToDo: To be deleted once markup has been updated to .video--wrapper
+.video-wrap,
```

Comment:
Is this still a todo - does `.video-wrap` need to become `.video--wrapper`, or are we having both, that do the same thing?

Reply:
It is still a TODO, however it felt inappropriate to leave a TODO comment within the file.